pom.xml

```diff
@@ -99,14 +99,14 @@
     <module>sql/hive</module>
     <module>unsafe</module>
     <module>assembly</module>
-    <module>external/twitter</module>
-    <module>external/flume</module>
-    <module>external/flume-sink</module>
-    <module>external/flume-assembly</module>
-    <module>external/mqtt</module>
-    <module>external/mqtt-assembly</module>
-    <module>external/zeromq</module>
-    <module>examples</module>
+    <!--<module>external/twitter</module>-->
```

Comment:
These changes only used by myself, don't need  merge. But I hope the following changes be merged:
Add `com.google.common.base.AbstractIterator` exceptions to Guava relocation , So far, all guava 14.0.1 `Optional` Class dependencies have added;